components/yaml/introduction.rst

```diff
@@ -219,6 +222,55 @@ representation to the inline one:
         foo: bar
         bar: baz
 
+Indentation
+"""""""""""
+
+By default the YAML component will use 4 spaces for indentation, this can be
+changed using the second argument as follows:
+
+.. code-block:: php
+
+    // use 8 spaces for indentation
```

Comment:
let's be less radical and use the 2 spaces indentation. This is closer to reality (Drupal for instance uses 2 spaces in Yaml).